Router.svelte

```diff
@@ -3,6 +3,24 @@ import {readable, writable, derived} from 'svelte/store'
 import {tick} from 'svelte'
 import {wrap as _wrap} from './wrap'
 
+
+
+async function runInSequence(functions, dispatchNextTick) {
```

Comment:
Can this run in parallel rather than sequence? This way props would be resolved in parallel and things would be faster